deep_tempering/callbacks.py

```diff
@@ -519,21 +521,26 @@ def exchange(self, **kwargs):
       beta_j = 1. / hyperparams[j]
 
     # beta_i - beta_j is expected to be negative
-    proba = min(np.exp(
-        coeff * (losses[i] - losses[j]) * (beta_i - beta_j)), 1.)
+    delta = self.coeff * (losses[i] - losses[j]) * (beta_i - beta_j)
+    proba = min(np.exp(delta), 1.)
 
     if np.random.uniform() < proba:
       swaped = 1
-      self.model.hpspace.swap_between(i, j, hpname)
+      self.model.hpspace.swap_between(replicas_ids[i], replicas_ids[j], hpname)
     else:
       swaped = 0
 
+    if getattr(self, 'exchange_logs', None):
+      accpt_ratio = (self.exchange_logs['swaped'].count(1) + swaped) / (len(self.exchange_logs['proba']) + 1)
+    else:
+      accpt_ratio = swaped
+
     super().log_exchange_metrics(losses, proba=proba, hpname=hpname,
-                                 swaped=swaped)
+                                 swaped=swaped, accept_ratio=accpt_ratio, delta=delta, exchange_pair=[replicas_ids[i], replicas_ids[j]])
```

Comment:
same here. please make sure that all lines are below conventional 80 characters.